.github/workflows/image-publish.yml

```diff
@@ -11,10 +11,12 @@ name: image-publish-insiders
 
```

Comment:
please fix the name of the workflow as it's not just insiders. Since other projects use a "release.yml" file, you could rename this one for consistency with those projects.